Dockerfile

```diff
@@ -3,7 +3,7 @@ RUN git clone https://github.com/newrelic/nrjmx.git && \
     cd nrjmx && \
     mvn package -DskipTests -P \!deb,\!rpm,\!test,\!tarball
 
-FROM golang:1.22 as builder
+FROM golang:1.22.7 as builder
```

Comment:
i would perhaps leave minor pinned to be consistant with the other repos owned by caos